### DISCUSSION TITLE
20倍速度提升：RUST 实现的 POSTGRES 向量相似度插件 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#19] [General]
# 20倍速度提升：RUST 实现的 POSTGRES 向量相似度插件 | 高策

江湖小虾米

http://gaocegege.com/Blog/pgvectors

**Comment by kun-teng** [comment]
Terrific for 20X.
But is it tested on GPU or CPU?

**Reply by gaocegege** [reply]
Only on CPUs
